fullcalendarComponent$demo/.project

```diff
@@ -3,12 +3,11 @@
 	<name>fullcalendarComponent$demo</name>
 	<comment></comment>
 	<projects>
-		<project>resources</project>
-		<project>fullcalendar_localhost_resources</project>
+		<project>fullcalendar_resources</project>
```

Comment:
Resource project fullcalendar_resources is indeed the one to be used in sample demo of fullcalendar, however the fullcalendar is used also by sampleGallery (demo.servoy.com) and it needs to share the same default "resources" project

Reply:
How to have two different resources projects?